pyproject.toml

```diff
@@ -1,3 +1,3 @@
 [build-system]
-requires = ["setuptools", "Cython"]
+requires = ["setuptools", "Cython", "pkgconfig"]
```

Comment:
This is not needed anymore, right?

```suggestion
requires = ["setuptools", "Cython"]
```

Reply:
right, a leftover from an intermediate version